pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -292,7 +302,7 @@ loki.write "default" {
 	endpoint {
 		url                = env("{{ .LokiURLEnvVarName }}")
 		max_backoff_period = "{{ .MaxBackoffPeriod }}"
-		tenant_id          = env("{{ .TenantIDEnvVarName }}")
+		tenant_id          = "giantswarm"
```

Comment:
I liked the idea of using an env var to set the default tenant, why removing this ?

Reply:
What value does it bring?